simpleclient_guava/src/main/java/io/prometheus/client/cache/guava/CacheMetricsCollector.java

```diff
@@ -0,0 +1,79 @@
+package io.prometheus.client.cache.guava;
```

Comment:
guava is a lot more than a cache, we might want to do guave.cache to allow for future stuff